test/e2e/create_cluster_test.go

```diff
@@ -56,6 +56,47 @@ func TestAWSCreateCluster(t *testing.T) {
 	e2eutil.EnsureNoCrashingPods(t, ctx, client, hostedCluster)
 }
 
+// TestKubeVirtCreateCluster implements a test that mimics the operation described in the
+// HyperShift quick start (creating a basic guest cluster).
+//
+// This test is meant to provide a first, fast signal to detect regression; it
+// is recommended to use it as a PR blocker test.
+func TestKubeVirtCreateCluster(t *testing.T) {
```

Comment:
Could we document what expecations for the management cluster we have so this works? I.E. how do you install kubevirt (is it installed by default in OCP?) and that it needs nodes that support KVM?

Reply:
Agreed, We need a commit/separate PR to properly introduce kubevirt docs ([same for agent platform](https://github.com/openshift/hypershift/pull/808#issuecomment-1004216829))

Reply:
> Could we document what expecations for the management cluster we have so this works? I.E. how do you install kubevirt (is it installed by default in OCP?) and that it needs nodes that support KVM?

yep, @nirarg I believe this is something you're working on?